third_party/move/move-compiler-v2/tests/bytecode-generator/v1-typing/global_invalid.exp

```diff
@@ -18,23 +18,23 @@ module 0x42::m {
 
 Diagnostics:
 error: Expected a struct type. Global storage operations are restricted to struct types declared in the current module. Found: '#0'
-  ┌─ tests/bytecode-generator/global_invalid.move:4:17
+  ┌─ tests/bytecode-generator/v1-typing/global_invalid.move:4:17
   │
 4 │         assert!(exists<T>(addr), 0);
   │                 ^^^^^^^^^^^^^^^
   │                 │
   │                 Invalid call to exists<#0>.
```

Comment:
I think it is the because `T` is a generic type that is not instantiated with a concrete type here. Or you mean we should use "T" instead of "#0" in the error message?

Reply:
Yes, `#0` is a bad message.  Better to show `T`.

Reply:
A bug is filed: https://github.com/aptos-labs/aptos-core/issues/12116

Reply:
Fixed in A fix is added to this PR: https://github.com/aptos-labs/aptos-core/pull/12085/commits/a7d67e1937300acbfaf27485f1cb5cafc7f81ce2